spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -221,15 +221,7 @@ public boolean pushAggregation(Aggregation aggregation) {
       return false;
     }
 
-    TableScan scan = table.newScan().includeColumnStats();
-    Snapshot snapshot = readSnapshot();
-    if (snapshot == null) {
-      LOG.info("Skipping aggregate pushdown: table snapshot is null");
-      return false;
-    }
-    scan = scan.useSnapshot(snapshot.snapshotId());
-    scan = configureSplitPlanning(scan);
-    scan = scan.filter(filterExpression());
+    org.apache.iceberg.Scan scan = buildIcebergBatchScan(true);
```

Comment:
Please modify boolean parameters you are passing with an inline comment to follow the [Contributing Style Guidelines on Boolean Arguments](https://github.com/apache/iceberg/blob/master/CONTRIBUTING.md#boolean-arguments)

Reply:
Fixed. Thanks